webapp/web/configuration_form.css

```
@@ -2,64 +2,103 @@
     display: grid;
     grid-template-rows: auto;
     grid-gap: 20px;
-    border: 1px solid #ddd;
-    border-radius: .5rem;
     padding: 20px;
 }
 
-.single-form-group {
-    display: flex;
-    padding: 27px 0px 27px 10px;
-    border-radius: .5rem;
-    box-shadow: 0px 0px 3px rgba(0, 0, 0, 0.22);
+.form-group {
+  display: grid;
+  grid-gap: 10px;
 }
 
-.single-form-group--in-multi {
-    border: none;
-    border-radius: none;
-    box-shadow: none;
-    padding: 10px 0px;
+.form-group__label {
+  margin-bottom: 10px;
+  font-weight: 500;
+  white-space: nowrap;
 }
 
-.single-form-group__label {
-    align-self: center;
-    padding-right: 10px;
-    font-size: 1.1em;
+.form-group-item {
+  display: grid;
+  grid-template-columns: 25% 40%;
```

If fixing the width, these should add up to 100%:
```suggestion
  grid-template-columns: 30% 70%;
```